### PR TITLE
CI: skip codecov if $CODE_COV_TOKEN is not defined

### PR DESCRIPTION
**Pull request by justinmk3**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ script:
 - ./gradlew check -S --info --console plain
 
 after_success:
-- bash <(curl -s https://codecov.io/bash) -F unittest
+- test $TRAVIS_BRANCH = "master" && bash <(curl -s https://codecov.io/bash) -F unittest
```
Comment (justinmk3) [inline .travis.yml]: Should change this to check for `CODE_COV_TOKEN` later. But it works for our purposes meanwhile.

**Comment by abrooksv** [comment]
I am not seeing the benefit of these changes, what is warranting it?

**Comment by justinmk3** [comment]
> I am not seeing the benefit of these changes, what is warranting it?

Avoids unnecessary downloads to third parties, which avoids potential failed builds.

File: buildspec/windowsTests.yml [hunk 2]
```diff
@@ -14,7 +14,10 @@ env:
 phases:
   install:
     commands:
-      - choco install -y --no-progress codecov
+      - |
+        if(-Not($Env:CODEBUILD_BUILD_SUCCEEDING -eq "0" -Or $Env:CODE_COV_TOKEN -eq $null)) {
```
Comment (rli) [inline buildspec/windowsTests.yml]: Since this is the first command in the phase, isn't `CODEBUILD_BUILD_SUCCEEDING` always true?

Comment (justinmk3) [reply buildspec/windowsTests.yml]: I wasn't sure where that env var actually gets defined. Will remove it from here.

**Comment by justinmk3** [comment]
Anything blocking this?

File: .travis.yml (hunk 1 above)
Comment (abrooksv) [inline .travis.yml]: So we no longer will do CodeCov in Travis PRs?

Comment (justinmk3) [reply .travis.yml]: TRAVIS_BRANCH is the target branch

Comment (abrooksv) [reply .travis.yml]: Ah

**Pull request merged**